src/borg/helpers/parseformat.py

```diff
@@ -405,6 +414,16 @@ def normpath_special(p):
             p = os.path.normpath(p)
             return ('/.' + p) if relative else p
 
+        m = self.win_file_re.match(text)
+        if m:
+            self.proto = 'file'
+            self.path = os.path.normpath(m.group('path'))
+            self.archive = m.group('archive')
+            # Only consider the parsing complete if the path is not the current directory.
```

Comment:
add why.

Reply:
This is no longer required. I found out this was needed because `win_file_re` also matched empty strings. I fixed it in the regex.